tests/test_models/test_dense_heads/test_mask2former_head.py

```diff
@@ -11,10 +11,12 @@ def test_mask2former_head_loss():
     base_channels = 64
     img_metas = [{
         'batch_input_shape': (128, 160),
+        'pad_shape': (128, 160, 3),
```

Comment:
Need to design unit test for instance segmentation

Reply:
Reworked the unit test to test panoptic and instance segmentation.  Please take a look at [L62](https://github.com/PeterVennerstrom/mmdetection/blob/bf6bc241e9cae60f05557ab14f72ec1c0e00ce61/tests/test_models/test_dense_heads/test_mask2former_head.py#L62).  Added a variable which is 100 when testing panoptic and 80 when testing instance seg.  Want to make sure this is still a valid test for instance seg.